ml-agents/mlagents/trainers/sac/trainer.py

```diff
@@ -326,7 +324,7 @@ def add_policy(
                     self.__class__.__name__
                 )
             )
-        if not isinstance(policy, NNPolicy):
+        if not isinstance(policy, TFPolicy):
             raise RuntimeError("Non-SACPolicy passed to SACTrainer.add_policy()")
```

Comment:
We should change this error to TFPolicy. Or maybe we should just remove these checks - they were a lot more useful when we had multiple Policy types.

Reply:
Should we just check for the `Policy` base class?

Reply:
I think we should just put the type check in the method and not have this runtime error. We shouldn't be passing non-policy objects in the first place 😛

Reply:
Cool. I just got rid of these checks.